jenkins/scripts/files/run_clean.sh

```diff
@@ -6,14 +6,18 @@ REPO_NAME="${1:-metal3-dev-env}"
 IMAGE_OS="${2:-ubuntu}"
 TESTS_FOR="${3:-e2e_tests}"
 
+if [ "${REPO_NAME}" == "metal3-dev-env" ]; then
```

Comment:
I guess we do not need the if condition we can directly source the vars.sh file

Reply:
Well we can I was just thinking that in case of other repos it is not necessary . Although you are right in theory every repo that uses this shell script should have a "/tmp/vars.sh" so I guess I can remove the condition.